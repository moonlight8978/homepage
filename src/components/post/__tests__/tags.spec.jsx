import React from 'react'
import { render } from '@testing-library/react'

import Tags from '../tags'

test('renders tags when input is not empty', () => {
  const tags = ['JS', 'Ruby', 'Miscs']
  const { container } = render(<Tags values={tags} />)
  expect(container.firstChild).toMatchSnapshot()
})

test('does not render anything when input is empty', () => {
  const tags = []
  const { container } = render(<Tags values={tags} />)
  expect(container.firstChild).toBeNull()
})
