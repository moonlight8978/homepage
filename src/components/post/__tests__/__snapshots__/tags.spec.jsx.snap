// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders tags when input is not empty 1`] = `
<div
  class="tags"
>
  <div
    class="tag"
  >
    JS
  </div>
  <div
    class="tag"
  >
    Ruby
  </div>
  <div
    class="tag"
  >
    Miscs
  </div>
</div>
`;
